test/specs/apexReplayDebugger.e2e.ts

```diff
@@ -0,0 +1,172 @@
+    const successNotificationWasFound = await utilities.notificationIsPresent(workbench, 'SFDX: Turn On Apex Debug Log for Replay Debugger successfully ran');
+    expect(successNotificationWasFound).toBe(true);
+  });
+
+  step('Run the Anonymous Apex Debugger with Currently Selected Text', async () => {
+    // Get open text editor
+    const workbench = await browser.getWorkbench();
+    const editorView = workbench.getEditorView();
+
+    // Open test file
+    const textEditor = await editorView.openEditor('ExampleApexClassTest.cls') as TextEditor;
+
+    // Select text
+    await textEditor.selectText('ExampleApexClass.SayHello(\'Cody\');');
+    await utilities.pause(1);
+
+    // Run SFDX: Launch Apex Replay Debugger with Currently Selected Text.
+    await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Execute Anonymous Apex with Currently Selected Text', 1);
+
+    // Wait for the command to execute
+    await utilities.waitForNotificationToGoAway(workbench, 'Running Execute Anonymous Apex', fiveMinutes);
+
+    const successNotificationWasFound = await utilities.notificationIsPresent(workbench, 'Execute Anonymous Apex successfully ran');
+    expect(successNotificationWasFound).toBe(true);
+    await utilities.pause(1);
+  });
+
+  step('SFDX: Get Apex Debug Logs', async () => {
+    // Run SFDX: Get Apex Debug Logs
+    const workbench = await browser.getWorkbench();
+    prompt = await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Get Apex Debug Logs', 1);
+
+    // Select the "ExampleApexClassTest" file
+    await prompt.selectQuickPick('ExampleApexClassTest');
+
+    // Wait for the command to execute
+    await utilities.waitForNotificationToGoAway(workbench, 'Getting Apex debug logs', fiveMinutes);
+
+    // Select a log file
+    const quickPicks = await prompt.getQuickPicks();
+    expect(quickPicks).not.toBeUndefined();
+    expect(quickPicks.length).toBeGreaterThanOrEqual(1);
+    await prompt.selectQuickPick('User User - Api');
+
+    // Wait for the command to execute
+    await utilities.waitForNotificationToGoAway(workbench, 'Running SFDX: Get Apex Debug Logs', fiveMinutes);
+
+    const successNotificationWasFound = await utilities.notificationIsPresent(workbench, 'SFDX: Get Apex Debug Logs successfully ran');
+    expect(successNotificationWasFound).toBe(true);
+  });
+
+  step('SFDX: Launch Apex Replay Debugger with Last Log File', async () => {
+    // Run SFDX: Launch Apex Replay Debugger with Last Log File
+    const workbench = await browser.getWorkbench();
+    prompt = await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Launch Apex Replay Debugger with Last Log File', 1);
+
+    // Get open text editor
+    const editorView = await workbench.getEditorView();
+
+    // Get file path from open text editor
+    const activeTab = await editorView.getActiveTab();
+    if (!activeTab) {
```

Comment:
Are lines 108-110 needed?  Won't activeTab always be "not undefined"?

Reply:
I think I had that because I was getting like this tslint error that activeTab might be undefined, but yeah, it's not needed. Thanks!